bundles/org.eclipse.text.quicksearch/src/org/eclipse/text/quicksearch/internal/ui/QuickSearchDialog.java

```diff
@@ -239,7 +240,8 @@ public void update(ViewerCell cell) {
 
 	private Image getBlankImage() {
 		if (blankImage==null) {
-			blankImage = new Image(Display.getDefault(), 1, 1);
+			final ImageGcDrawer imageGcDrawer = (gc, width, height) -> {};
+			blankImage = new Image(Display.getDefault(), imageGcDrawer, 1, 1);
```

Comment:
This change alone doesn't look really beneficial.
if you want to consider the commented out code below, I suggest to move that code into the new lambda.